clang-15: `stdnoreturn.h` is deprecated in C2x

Can't build brutal using clang-15 since `<stdnoreturn.h>` issue a warning (due to deprecation) and brutal enable `-Werror`.

```
/usr/lib/llvm-15/lib/clang/15.0.0/include/stdnoreturn.h:24:2: error: "the '<stdnoreturn.h>' header is deprecated in C2x; either use the '_Noreturn' keyword or the '[[noreturn]]' attribute" [-Werror,-W#warnings]
#warning "the '<stdnoreturn.h>' header is deprecated in C2x; either use the '_Noreturn' keyword or the '[[noreturn]]' attribute"
 ^
```

--- a/sources/libs/brutal/base/std.h
+++ b/sources/libs/brutal/base/std.h
@@ -5,4 +5,5 @@
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
-#include <stdnoreturn.h>
+
+#define noreturn _Noreturn

